internal/commands/config/list_keys_internal.go

```diff
@@ -0,0 +1,106 @@
+package config_internal
+
+import (
+	"strings"
+
+	"github.com/pingidentity/pingcli/internal/configuration"
+	"github.com/pingidentity/pingcli/internal/configuration/options"
+	"github.com/pingidentity/pingcli/internal/logger"
+	"github.com/pingidentity/pingcli/internal/output"
+	"github.com/pingidentity/pingcli/internal/profiles"
+	"gopkg.in/yaml.v3"
+)
+
+func returnKeysYamlString() string {
+	inputString := returnKeysString("", " ")
+	// Split the input by spaces
+	parts := strings.Fields(inputString)
```

Comment:
I think this is just easier to call `configuration.ViperKeys()` and decouple to two functions here.

that way, the `returnKeysString` doesnt need arguments either